### DISCUSSION TITLE
Option to Assign a Skin to a Game Mode

### DISCUSSION THREAD

**Discussion by c0rty0** [ppy/osu#28775] [Ideas] [answered]
Whenever I want to switch game modes, I have to switch skins. I know this is lazy, but having a feature allowing the user to assign a certain skin to a certain game mode would be cool. The feature would be up to the user to use. It would not automatically be in use, but the user can turn it on to assign skins, so this feature would not be weird or something. I think a feature where switching your game mode switches your skins as well would be helpful.

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/28695. Please upvote that original discussion with already 7 upvotes.
